### ISSUE TITLE
Android API Level 17 Error

### ISSUE THREAD

**Issue by bellalMohamed** [gogoout/nativescript-rater#5]
Error while building android on api lever 17

```
/Users/bellalmohamed/Development/zeal-area/Zeal-Members-App-Native/platforms/android/app/src/main/AndroidManifest.xml Error:
        uses-sdk:minSdkVersion 17 cannot be smaller than version 21 declared in library [:nativescript-rater:] /Users/bellalmohamed/.gradle/caches/transforms-2/files-2.1/cb8dc1ca29e8e5d1b47ded14a870aa9d/nativescript-rater/AndroidManifest.xml as the library might be using APIs not available in 17
        Suggestion: use a compatible library with a minSdk of at most 17,
                or increase this project's minSdk version to at least 21,
                or use tools:overrideLibrary="hotchemi.android.rate" to force usage (may lead to runtime failures)
```

**Comment by gogoout** [comment]
It's part of the changes in https://github.com/gogoout/nativescript-rater/pull/2

**Comment by mcalc001** [comment]
Hey, we're experiencing this same issue are there any updates on this fix? Thanks!

**Comment by gogoout** [comment]
Unfortunately, I no longer maintain this plugin any more, but I'll be happy to accept any merge.

**Comment by igorkra** [comment]
Should be fixed in #15 
@gogoout Could you please update npm version?

**Comment by gogoout** [comment]
@igorkra Done. Check 3.0.0 on npm. Let me know if it works.
